### PR TITLE
Add Sort Option Translation & disable sort that is not fast sort

### PR DESCRIPTION
**Pull request by taooceros**
Everything is indexing everything, so a sort that is not fast sort can cost a really long time to run.

### CODE REVIEW

**Comment by jjw24** [comment]
Version bump please

**Comment by jjw24** [comment]
What is this fast sort option?

**Comment by taooceros** [comment]
> What is this fast sort option?

Index from everything

**Comment by taooceros** [comment]
image

**Comment by jjw24** [comment]
oh ok, it's a setting from Everything

File: Everything/EverythingAPI.cs [hunk 1]
```diff
@@ -123,13 +123,18 @@ public List<SearchResult> Search(string keyWord, CancellationToken token, SortOp
                 EverythingApiDllImport.Everything_SetSearchW(keyWord);
                 EverythingApiDllImport.Everything_SetOffset(offset);
                 EverythingApiDllImport.Everything_SetMax(maxCount);
+                
+                if(!EverythingApiDllImport.Everything_IsFastSort(sortOption))
+                {
+                    throw new InvalidOperationException("The Sort Option is not Fast Sort, it may take a long time to finish the query");
```
Comment (jjw24) [inline Everything/EverythingAPI.cs:129]: should this be a prompt instead?

Comment (taooceros) [reply Everything/EverythingAPI.cs:129]: It will be displayed like this
image

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
it's actually not disabling sort though right when it's not fast sort?

**Comment by taooceros** [comment]
It will disable search when sort is not fast sort, prompting users to choose a fast sort option.

**Pull request merged**